pkg/didcomm/common/service/service.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package service
+
+// Handler provides protocol service handle api.
+type Handler interface {
+	Handle(msg DIDCommMsg) error
+}
+
+// DIDCommService defines service APIs.
+type DIDCommService interface {
```

Comment:
Consider removing the "Service" suffix since this is now in the `service` package.